src/test/java/pl/coderslab/imageManager/image/ImageControllerTest.java

```diff
@@ -1,39 +1,40 @@
-package pl.coderslab.imageManager.controllers;
+package pl.coderslab.imageManager.image;
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
 import org.springframework.test.web.servlet.setup.MockMvcBuilders;
-import pl.coderslab.imageManager.repositories.ImageRepository;
-import pl.coderslab.imageManager.services.ImageService;
+import pl.coderslab.imageManager.mock.MockDatabase;
 
 import static org.mockito.Mockito.mock;
 
 class ImageControllerTest {
     private static ImageController imageController;
-    private static String HOME_PAGE_VIEW_NAME = "home";
-    private static String FORM_PAGE_VIEW_NAME = "form";
-    private ImageService imageService;
+    private static final String HOME_PAGE_VIEW_NAME = "home";
+    private static final String FORM_PAGE_VIEW_NAME = "form";
+    private static ImageFacade imageFacade;
+    private static ImageRepository imageRepository;
 
     @BeforeEach
-    public void beforeClass() {
-        imageService = new ImageService(mock(ImageRepository.class));
-        imageController = new ImageController(imageService);
+    public void setUp() {
+        imageRepository = new MockDatabase();
```

Comment:
tutaj nie musisz robić bazy dlatego że to są testy kontrolera,  czyli testujesz tylko i wyłącznie ten jeden wycinek apki - kontroler :p . Baza nie jest potrzebna = może być mock